Formula/minicom.rb

```diff
@@ -20,15 +21,19 @@ class Minicom < Formula
     sha256 x86_64_linux:   "7d1b0aae1f169968d42e4dea644dff5a4f18010b59b334439aa2bd276c6e913a"
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "gettext"
+
   uses_from_macos "ncurses"
 
   def install
     # There is a silly bug in the Makefile where it forgets to link to iconv. Workaround below.
     ENV["LIBS"] = "-liconv" if OS.mac?
 
-    system "./configure", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}",
-                          "--mandir=#{man}"
+    system "./autogen.sh"
```

Comment:
```suggestion
    system "./autogen.sh" if build.head?
```
Restrict this to `--HEAD` builds so that we don't need the extra dependencies needed to run `autogen.sh`.

Reply:
yep, done with latest commit